maven-model/src/main/mdo/maven.mdo

```diff
@@ -3459,6 +3459,7 @@
           <type>String</type>
           <defaultValue>2.0</defaultValue>
           <description><![CDATA[
+            <b>Deprecated</b>. Use the Maven Enforcer Plugin's <a href="https://maven.apache.org/enforcer/enforcer-rules/requireMavenVersion.html"><code>requireMavenVersion</code></a> rule instead.<br>
```

Comment:
This <prerequisites> is still supported for Maven Plugins (https://issues.apache.org/jira/browse/MNG-5501?focusedCommentId=14419813&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-14419813), therefore rather than deprecating that I would add a hint, that this should only be used for packaging "plugin".